src/Analyzer.Cli.FunctionalTests/Analyzer.Cli.FunctionalTests.csproj

```diff
@@ -28,6 +28,9 @@
     <None Update="Tests\Invalid.bicep">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Tests\ToTestSeparateParametersFile\TemplateWithSeparateParametersFile.json">
```

Comment:
The whole Tests folder is copied on line 22, can we revert this change?

Reply:
Weird, I don't remember setting that config. Anyway, fixed in [`d3dd96d` (#297)](https://github.com/Azure/template-analyzer/pull/297/commits/d3dd96d32e7b79c8c7bd89fe54d9909a95c53700) 😄